sanicbot/core/github.py

```diff
@@ -1,61 +1,51 @@
 import re
 
-import httpx
+from aiohttp import ClientSession
 from discord import Message
 from discord.ext import commands
-from discord.ext.commands import Context
-
 from sanicbot.core.utils import failure_message, success_message
 
 
 class GitCog(commands.Cog):
     issue_pattern = re.compile(r"#(?P<issue_id>[1,2]\d{3})")
 
-    def __init__(self, bot):
+    def __init__(self, bot: commands.Bot):
         self.bot = bot
+    
+    @commands.Cog.listener()    # no where else to throw this so lets put it here
+    async def on_ready(self):
+        print(f'Logged in as {self.bot.user.name}')
```

Comment:
Why print?

Reply:
I believe this was just part of the quickstart documentation. I have nearly identical code in a personal bot of mine

Reply:
Let's get rid of that. If we want to output anything, we should use the `logging` module.